locale/moretrees.es.tr

```diff
@@ -211,10 +211,10 @@ Willow Tree Trunk Stair=Escalera de Tronco de Árbol de Sauce
 Yellow=Amarillo
 
 # init.lua
-Sapling=
-Tree=
-Wooden Planks=
-Leaves=
-Wooden Fence=
-Wooden Fence Rail=
-Wooden Fence Gate=
+Sapling=Retoño
+Tree=Puu
```

Comment:
I have never heard of "Puu" and neither does RAE know about this. "Árbol" is the general word for tree.